website/docs/docs/collaborate/cloud-build-and-view-your-docs.md

```diff
@@ -19,6 +19,15 @@ To set up a job to generate docs:
 
 4. Click **Save**. Proceed to [configure project documentation](#configure-project-documentation) so your project generates the documentation when this job runs.
 
+You can also add `dbt docs generate` to the list of commands in the job run steps. However, you can expect different outcomes when you configure a job using the **Generate docs on run** selection, compared to when you add the command to the run steps. 
+
+Review the following options and outcomes:
+
+| Options | Outcomes |
+|--------| ------- |
+| **1. Select checkbox** | Select the **Generate docs on run** checkbox to automatically generate updated project docs each time your job runs. If that particular step in your job fails, the job can still be successful if all subsequent steps are successful. |
```

Comment:
They don't need to do both of these steps in this order, right? If not then I suggest removing step numbers which might be misleading.
```suggestion
| **Select checkbox** | Select the **Generate docs on run** checkbox to automatically generate updated project docs each time your job runs. If that particular step in your job fails, the job can still be successful if all subsequent steps are successful. |
```